app/presenters/user_analytics_presenter.rb

```diff
@@ -146,7 +147,7 @@ def last_updated_at
 
   def statistics
     @statistics ||=
-      Rails.cache.fetch(statistics_cache_key, expires_in: EXPIRE_STATISTICS_CACHE_IN) {
+      Rails.cache.fetch(statistics_cache_key, expires_in: EXPIRE_STATISTICS_CACHE_IN, force: bust_cache?) {
```

Comment:
Added this to help with debugging

Reply:
We also need to update the user analytics controller to _set_ `RequestStore[:bust_cache]`, right?

Reply:
Hrm right, this does not inherit from `AdminController`. 👍🏼